### PR TITLE
Add blog post for cycle 7 results

### PR DESCRIPTION
**Pull request by m52go**

### CODE REVIEW

File: _posts/2019-11-22-cycle-7-results.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+author: Steve Jain
+excerpt: "Cycle 7 of the Bisq DAO ended at block 604 506 on November 18 2019. This post covers its results. <br><br>"
+en-only: true
+---
+
+This post summarizes the results of Cycle 7 of the Bisq DAO.
+
+### Summary
+
+* Cycle took place between blocks 604 507 and 609 186
```
Comment (devinbileck) [inline _posts/2019-11-22-cycle-7-results.md]: Cycle 7 was blocks 599 827 to 604 506

File: _posts/2019-11-22-cycle-7-results.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Bisq DAO Cycle 7: Results"
+author: Steve Jain
+excerpt: "Cycle 7 of the Bisq DAO ended at block 604 506 on November 18 2019. This post covers its results. <br><br>"
```
Comment (devinbileck) [inline _posts/2019-11-22-cycle-7-results.md]: It ended on November 19

File: _posts/2019-11-22-cycle-7-results.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Ultimately, the risk was acknowledged but not considered an urgent issue.
+
+1. Although the donation address holder is outwardly anonymous, the person who holds the role was voted into it by sufficient BSQ weight, indicating core contributors are confident this person is trustworthy.
+2. BTC and BSQ transactions are public, so in the event there are issues, the person would be outed quickly, lose their bond, and probably become an outcast.
+
+While this not what people wanted to hear, it's possible expectations for the new trade protocol were set too high. The new trade protocol does not promise _trustless_ trading and dispute resolution—such a thing may not even be possible. It does, however, _reduce_ trust in some places (no anonymous figure with third key) and _decentralize_ trust in others (responsibility to make traders whole is now spread across the counterparty, mediator, arbitrator, and the Bisq DAO).
```
Comment (devinbileck) [inline _posts/2019-11-22-cycle-7-results.md]:
```suggestion
While this is not what people wanted to hear, it's possible expectations for the new trade protocol were set too high. The new trade protocol does not promise _trustless_ trading and dispute resolution—such a thing may not even be possible. It does, however, _reduce_ trust in some places (no anonymous figure with third key) and _decentralize_ trust in others (responsibility to make traders whole is now spread across the counterparty, mediator, arbitrator, and the Bisq DAO).
```

File: _posts/2019-11-22-cycle-7-results.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+And of course, liability on the network itself is reduced signficantly without the third key.
```
Comment (devinbileck) [inline _posts/2019-11-22-cycle-7-results.md]:
```suggestion
And of course, liability on the network itself is reduced significantly without the third key.
```

**Review by devinbileck — CHANGES_REQUESTED** [review]
Aside from a few corrections, looks good.

**Comment by m52go** [comment]
Thanks @devinbileck for the keen eye, not sure why I keep getting these dates and blocks wrong.

**Pull request merged**